### PR TITLE
Implement Storage class

### LINKED ISSUE #50: Implement storage functionality
Implement storage functionality to save events locally

### PR DESCRIPTION
**Pull request by KuanHsienn**
Add functionality to save current events locally to a text file - "events.txt"

Fixes #50

### CODE REVIEW

File: src/main/java/seedu/manager/Main.java [hunk 1]
```diff
@@ -5,17 +5,24 @@
...
 import seedu.manager.parser.Parser;
 import seedu.manager.ui.Ui;
+import seedu.manager.storage.Storage;
+
+import java.io.IOException;
 
 public class Main {
     private static final Ui ui = new Ui();
     private static EventList events = new EventList();
+    private static final String filePath = "events.txt";
```
Comment (jemehgoh) [inline src/main/java/seedu/manager/Main.java:15]: This static variable seems to me to be a constant, so I think it should be named following the convention for constants?

File: src/main/java/seedu/manager/Main.java [hunk 2]
```diff
@@ -41,4 +48,28 @@ private static void runCommandLoop() {
...
+    /**
+     * Loads events from file and handles exceptions.
+     */
+    private static void loadData() {
+        try {
+            storage.loadEvents(events); // Load events using storage instance
+            ui.showMessage("Events loaded successfully.");
+        } catch (IOException e) {
+            ui.showErrorMessageToUser(e); // Use showErrorMessageToUser
+        }
```
Comment (jemehgoh) [inline src/main/java/seedu/manager/Main.java:61]: I think these comments are redundant and can be removed, as it is obvious enough to me what the code is doing?

File: src/main/java/seedu/manager/storage/Storage.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            for (String line : Files.readAllLines(Paths.get(filePath))) {
+                String[] parts = line.split(","); // CSV format
+                if (parts.length == 3) {
+                    String eventName = parts[0].trim();
+                    String time = parts[1].trim();
+                    String venue = parts[2].trim();
+                    events.addEvent(eventName, time, venue);
+                }
+            }
+        } catch (IOException e) {
```
Comment (jemehgoh) [inline src/main/java/seedu/manager/storage/Storage.java:61]: I think you should use a more descriptive name for the exception in this catch block?

**Review by jemehgoh — COMMENTED** [review]
Mostly looks good, just some nits.

File: src/main/java/seedu/manager/storage/Storage.java (hunk 3 above)
Comment (KuanHsienn) [reply src/main/java/seedu/manager/storage/Storage.java:61]: As in the error message?

Comment (jemehgoh) [reply src/main/java/seedu/manager/storage/Storage.java:61]: As in I think you should give the exception you are naming a more descriptive name than "e".

**Review by jemehgoh — APPROVED** [review]
Looks good to merge!

**Pull request merged**